### ISSUE TITLE
incorrect prediction of fusion junction sequences

### ISSUE THREAD

**Issue by ndaniel** [ndaniel/fusioncatcher#52]
In some rare cases that sequence of the fusion junction is predicted incorrectly even that the genomic coordinates of the fusion are correct.

An incorrect sequence of fusion junction (from file `final-list_candidate-fusion-genes.txt`) looks like this:
```
AAAACCACAATGAGATACCATCTCACACCAGTTAGAATGGTGATCATTAA*AACCACAATGAGATACCATCTCACACCAGTTAGAATGGTGATCATTAA
``` 
which is actually looking like duplicated sequence:
```
AAAACCACAATGAGATACCATCTCACACCAGTTAGAATGGTGATCATTAA*
  AACCACAATGAGATACCATCTCACACCAGTTAGAATGGTGATCATTAA
```

**Comment by ndaniel** [comment]
This will be fixed in the next release of FusionCatcher that is v0.99.7c.

Until then one may fix it manually, by changing line number 68 of file `smoothing_fusions_psl.py` from 
```
        fs2 = fs1[w:]
```
into
```
        fs2 = fs2[w:]
```

**Comment by ndaniel** [comment]
This will be fixed in v0.99.7c.
